programmers/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,13 @@
+function solution(numbers) {
+    let numbersum = [];
+    for(let a = 0; a < numbers.length; a++) {
+        for(let b = 0; b < numbers.length; b++) {
+            if(a != b) {
```

Comment:
js에서는 비교를 2개 (!=)가 아닌 3개 (!==)를 사용한다.
```suggestion
            if(a !== b) {
```